internal/util/test/controller_manager.go

```diff
@@ -99,7 +99,8 @@ func DeployControllerManagerForCluster(
 		defer os.Remove(kubeconfig.Name())
 		fmt.Fprintf(os.Stderr, "INFO: Starting Controller Manager for Cluster %s with Configuration: %+v\n", cluster.Name(), config)
 		if err := rootcmd.RunWithLogger(ctx, &config, logger); err != nil {
-			panic(err)
+			fmt.Fprintf(os.Stderr, "ERROR: Problems with Controller Manager: %s\n", err)
```

Comment:
This is actually a change I wouldn't want to introduce: having a log message in a different format makes parsing it difficult (think about log ingestions systems where one might have alerts on field existence or values). How would you then parse and alert on it? Why not just use `logger` with its e.g. `Error()` method 🤔 ?

Reply:
This code runs only in our CI system when you run integration tests and this is the format in which we log. When I started to make this change I thought by mistake that it was a user-facing code, but it's not. As I described in https://github.com/Kong/kubernetes-ingress-controller/pull/4952#discussion_r1369964228

Reply:
Fair enough. Regardless, I'd use the same logger as available in the scope of this func. I'm  not sure what's the benefit of using a different one (or `fmt.Fprintf`)

Reply:
Our helpers in tests everywhere else log in such format e.g. [here](https://github.com/Kong/kubernetes-ingress-controller/blob/3e0ff25ba3895b618bbdf7b80c09f383afd731c4/internal/util/test/crds.go#L24-L38), so it just consistency. Btw. maybe we should introduce a logger for test helpers, even a global one to use it instead of Print, etc.

Reply:
The link above is a 404 :)

Reply:
link is fixed vscode tricked me

Reply:
Alright then, let's do it this way and think about changing the logger in tests separately.